corporate/tests/test_stripe.py

```diff
@@ -5587,3 +5605,137 @@ def assert_audit_log(
             assert_audit_log(
                 audit_log, None, support_admin, audit_log_class.CUSTOMER_PLAN_CREATED, event_time
             )
+
+
+@override_settings(PUSH_NOTIFICATION_BOUNCER_URL="https://push.zulip.org.example.com")
+class TestRemoteRealmBillingFlow(StripeTestCase, BouncerTestCase):
```

Comment:
Should we inherit`RemoteBillingAuthenticationTest` instead of  `BouncerTestCase` so that we call `self.execute_remote_billing_authentication_flow` to login user?

Reply:
I guess we can extract a short version of it in stripe for testing so that we don't end up running the whole `RemoteBillingAuthenticationTest` test.

Reply:
Yeah, The reason I didn't inherit it from `RemoteBillingAuthenticationTest` was to not end up running the wholw thing. But abstracting within `RemoteBillingAuthenticationTest` makes sense. Making that change.

Reply:
This was done.